### PR TITLE
Translation carousel home

### LINKED ISSUE #184: translation for carousel in home page
fix translation in carousel home page

### PR DESCRIPTION
**Pull request by mohammad-jundi**
fixed the translation problem on the carousel home page 
#

### CODE REVIEW

**Review by jennetturkkan — APPROVED** [review]

**Comment by jennetturkkan** [comment]
@louisrli can you check this one too?

File: src/components/BlogsCarouselHome/index.jsx [hunk 1]
```diff
@@ -8,24 +8,40 @@ import "./index.scss";
 import "slick-carousel/slick/slick.css";
 import "slick-carousel/slick/slick-theme.css";
 import SliderArrow from "../SliderArrow";
+import i18next from "i18next";
 
-const POSTS_PER_PAGE = 9;
+const POSTS_PER_PAGE = 18;
```
Comment (louisrli) [inline src/components/BlogsCarouselHome/index.jsx]: actually...

https://developer.wordpress.org/rest-api/reference/posts/#list-posts

you guys should be using the "tags" param in the API. and when the language switches, you should be calling fetch() again.

its wrong for this to say 18 POSTS_PER_PAGE. are there 18 posts per page? nope

Comment (louisrli) [reply src/components/BlogsCarouselHome/index.jsx]: fix in the other PR too

**Comment by jennetturkkan** [comment]
@louisrli fixed this one too. can you check?

**Review by louisrli — APPROVED** [review]

File: src/components/BlogsCarouselHome/index.jsx (hunk 1 above)
Comment (jennetturkkan) [reply src/components/BlogsCarouselHome/index.jsx]: i don't see it locally :/

Comment (jennetturkkan) [reply src/components/BlogsCarouselHome/index.jsx]: removed

**Comment by jennetturkkan** [comment]
@Ammar-64 ready to be merged

**Pull request merged**